apps/staking/app/mystakes/modules/TestnetPointsModule.tsx

```diff
@@ -69,6 +69,7 @@ export default function TestnetPointsModule(params?: { addressOverride?: Address
       <ModuleDynamicQueryText
         status={status as QUERY_STATUS}
         fallback={0}
+        enabled
```

Comment:
I know this is allowed but I hate it 😂

Reply:
I am a purist enabled={true}. Just need to say something but you can ignore

Reply:
haahhaha this is where we differ, im a minimalist so im strongly againsts `enabled={true}` in my mind its like doing:
```ts
if (enabled == true){
  return null;
}
```

When you could just do:
```ts
if (enabled){
  return null;
}
```

Reply:
Interesting I see your viewpoint. Resolving